frontend/src/components/orders/SubOrderItemButton.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import Button from "primevue/button";
+import { ref } from "vue";
+
+import { createDocumentBySubOrder } from "@/backendClient";
+import useConfirmations from "@/compositions/useConfirmations";
+import useNotifications from "@/compositions/useNotifications";
+import { DocumentKind } from "@/global/types/appTypes";
+import {
+  type InvoiceDocument,
+  type OfferDocument,
+  type OverdueNoticeDocument,
+} from "@/global/types/entities";
+
+const props = defineProps<{
+  id: number;
+  kind: DocumentKind;
+}>();
+const confirm = useConfirmations();
+const notifications = useNotifications();
+
+let documents = ref<OfferDocument | OverdueNoticeDocument | InvoiceDocument>();
+async function createDocument() {
+  documents.value = await createDocumentBySubOrder(props.id, props.kind);
+  console.log(documents.value);
```

Comment:
Can you please do a code search and remove all console logs that should not be there?